util/test/sub_test

```diff
@@ -798,6 +798,8 @@ if (os.getenv('CHPL_TEST_VGRND_EXE')=='on' and vgrbin):
     valgrindbinopts = chpl_valgrind_opts.split()+['-q']
     if (chplcomm!='none'):
         valgrindbinopts+=['--trace-children=yes']
+        if (chplcomm=='gasnet'):
+            valgrindbinopts+=['--suppressions=%s/third-party/gasnet/gasnet-src/other/contrib/gasnet.supp'%(chpl_home)]
```

Comment:
Nit. sub_test has a hodgepodge of formatters so it doesn't really matter, but the new/preferred way to format strings in python is with .format():
```python
valgrindbinopts+=['--suppressions={0}/third-party/gasnet/gasnet-src/other/contrib/gasnet.supp'.format(chpl_home)]

```

Reply:
OK.  Not knowing a thing about Python, I just pattern matched off of line 789.  Do you want me to change it/them?

Reply:
Up to you. sub_test is enough of a mismatch that I don't mind either way, was more of an fyi then something that needs to be fixed.